WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -46,6 +48,60 @@ wca.cancelPendingAjaxAndAjax = function(id, options) {
   return wca._pendingAjaxById[id];
 };
 
+$.fn.competitionsMap = function(competitions) {
+  var $map = new google.maps.Map(document.getElementById(this.attr('id')), {
+    zoom: 2,
+    center: {lat: 0, lng: 0},
+    scrollwheel: true
+  });
+
+  var markers = [];
+
+  competitions.forEach(function(c) {
+
+    var contentString = "<a href=" + c.url + ">" + c.name + "</a><br />" + c.marker_date + " - " + c.cityName;
+
+    var infowindow = new google.maps.InfoWindow({
+      content: contentString
+    });
+
+    if (c.is_over) {
+      iconImage = 'https://maps.google.com/mapfiles/ms/icons/blue.png';
+    } else {
+      iconImage = 'https://maps.google.com/mapfiles/ms/icons/red.png';
+    }
+
+    c.marker = new google.maps.Marker({
+      map: $map,
+      position: {
+        lat: c.latitude_degrees,
+        lng: c.longitude_degrees,
+      },
+      title: c.name,
+      icon: iconImage
+    });
+
+    c.marker.addListener('click', function() {
+      infowindow.open($map, c.marker);
+    });
+
+    markers.push(c.marker);
+  });
+
+  var markerCluster = new MarkerClusterer($map, markers, {
+    maxZoom: 10,
+    clusterSize: 30
+  });
+};
+
+function isMobile() {
+  if(ResponsiveBootstrapToolkit.is('<sm')) {
```

Comment:
Also you can try my lib on moble, it will work,  but just not very well. It depends on the mobile browser and whether you use .responsive-table